src/main/webapp/app/entities/feedback.model.ts

```diff
@@ -33,6 +35,13 @@ export class Feedback implements BaseEntity {
     public referenceType: string | null; // this string needs to follow UMLModelElementType in Apollon in typings.d.ts
     public referenceId: string | null;
 
+    public static isStaticCodeAnalysisFeedback(that: Feedback): boolean {
```

Comment:
why do you call it `that` and not `feedback`?
Would it not make more sense to have this as instance method (without `static`) and check it for `this`

Reply:
That's what I first tried when I've written the method. The problem is that we don't get proper feedback instances from the server but just plain JavaScript Objects. Those objects don't have the proper prototype. I would need to create Feedback instances from the plain JSON received from the server. That's why all the other Feedback methods are also static.